File: RestroomFinder/Native/Apple/SharedFramework/Speed.swift
//
//  Speed.swift
//  SharedFramework
//
//  Created by DevTeam on 5/4/16.
//  Copyright © 2016 DevTeam. All rights reserved.
//

import Foundation
import MapKit
open class Speed
{
    open static func meterPesSecondToMPH(_ speed: CLLocationSpeed)-> Int{
        let meterPerHour = speed * 3600
        let kPerHour = meterPerHour / 1000
        let res = Int(round(kPerHour * 0.621371))
        return (res > 0 ? res : 0)
    }
    
    open static func distanceToMile(_ distance: CLLocationDistance) -> Double{
        let temp = NSString(format: "%.1f", distance * 0.000621371) as String
        return Double(temp)!
    }
}

File: RestroomFinder/Native/Android/RestroomFinder/app/src/main/java/org/actransit/restroomfinder/Infrastructure/Speed.java
package org.actransit.restroomfinder.Infrastructure;

/**
 * Created by DevTeam on 6/29/16.
 */
public class Speed
{
    public static int meterPesSecondToMPH(float speed){
        float meterPerHour = speed * 3600;
        float kPerHour = meterPerHour / 1000;
        int res = Math.round(kPerHour * 0.621371f);
        return (res > 0 ? res : 0);
    }

    public static double distanceToMile(float distance){
        String temp = String.format("%.1f",distance * 0.000621371);
        return Double.valueOf(temp);
    }
}
